src/main/java/com/bfa/core/page/GooglePage.java

```diff
@@ -0,0 +1,75 @@
+package com.bfa.core.page;
+
+import com.bfa.core.BrowserType;
+import com.bfa.core.util.BrowserUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+/**
+ * Created by ninhdoan on 12/24/16.
+ */
+public class GooglePage extends GenericPage {
+
+   private static final String GOOGLE_URL = "https://google.com";
+
+   private static final String SEARCH_BOX_NAME = "q";
+
+   private static final String SEARCH_RESULT_ID = "resultStats";
+
+   private static final String RESULT_LINK_XPATH = "//*[@id='rso']//h3/a";
+
+   private static final long DEFAULT_WAIT_TIME = 5;
```

Comment:
This is TIMEOUT and SECONDS. Please correct attribute name to reflect those meaning. Also, think of resource out this value because it could be used in many different code places.